### PR TITLE
Update docblocks on PHP methods

### PR DESCRIPTION
**Pull request by balbuf**

### CODE REVIEW

File: wp-remember-the-galleries/wp-remember-the-galleries.php [hunk 1]
```diff
@@ -111,6 +114,11 @@ static function action_init_post_type() {
 		) );
 	}
 
+	/**
+	 * Get attachment details for a set of IDs
```
Comment (bendoh) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: Can we put hard stops on each of the description parts?

Comment (bendoh) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: This would make it consistent with the first one.

Comment (balbuf) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: what do you think this is, Drupal?!?

Comment (balbuf) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: And yes, we can do that.

**Comment by balbuf** [comment]
Full stops in place. :hand:

**Comment by balbuf** [comment]
@bendoh you satisfied with this?

**Comment by bendoh** [comment]
Extremely! Do it! 🍓 /approved

**Pull request merged**